root/user/ReportUser.js

```diff
@@ -93,23 +93,18 @@ const ReportUser = ({
              {uri: {href: '/privileged', target: '_blank'}})}
     </p>
 
-    <p>
-      {l(`If you’d like us to be able to easily respond to your report,
-          please check “Reveal my email address” below.`)}
-    </p>
-
     <p>
       <strong>{addColon(l('Note'))}</strong>
       {' '}
       {l(`Be sure to provide direct links to examples of the behaviour you’re
-          reporting (for example, use “https://musicbrainz.org/edit/23”
-          instead of “edit #23”, and use “https://musicbrainz.org/edit/42 and
-          https://musicbrainz.org/edit/43” instead of
-          “https://musicbrainz.org/user/SomeUser/edits”).
-          Providing links makes it much easier for the recipients of the
-          report to look into the issues; a report without links is
-          unlikely to be acted on fast, since it will require a lot of
-          additional research.`)}
+        reporting (for example, use “https://musicbrainz.org/edit/23”
```

Comment:
Unexpected and erroneous indent change in https://github.com/metabrainz/musicbrainz-server/pull/951/commits/87f85b374f4ad97d2b28cb03fdb626f7f8480c50